services/venice-server/src/main/java/com/linkedin/venice/listener/ReadQuotaEnforcementHandler.java

```diff
@@ -427,8 +430,12 @@ public void handleStoreChanged(Store store) {
          *
          */
         this.onCustomizedViewChange(customizedViewRepository.getPartitionAssignments(topic));
+        if (versionNumber != currentVersion && versionNumber > backupVersion
```

Comment:
A safer condition would be "versionNumber < currentVersion" instead of "versionNumber != currentVersion"; WDYT?

Reply:
If we rollback the current version then it is valid for the new "backup" version to be greater than the current version right?

Reply:
IC! I am not sure if we would keep such "backup" version in the system. It's not a big concern, let me resolve this comment, but it would be great if we can double check.